src/applications/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -1,26 +1,30 @@
 /* eslint-disable jsx-a11y/label-has-for */
 import React from 'react';
 import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
 import recordEvent from 'platform/monitoring/record-event';
-import { updateBenefitSummaryRequestOption } from '../actions/letters';
+import { formatDateShort } from 'platform/utilities/date';
+import CallVBACenter from 'platform/static-data/CallVBACenter';
+
+import { updateBenefitSummaryRequestOption as updateBenefitSummaryRequestOptionAction } from '../actions/letters';
 import {
   benefitOptionsMap,
   characterOfServiceContent,
   optionsToAlwaysDisplay,
   getBenefitOptionText,
   stripOffTime,
-} from '../utils/helpers.jsx';
-import { formatDateShort } from 'platform/utilities/date';
-import CallVBACenter from 'platform/static-data/CallVBACenter';
+} from '../utils/helpers';
 
+/* eslint jsx-a11y/label-has-associated-control: 1 */
```

Comment:
@timwright12 Please see my comment "The second warning never went away" about why I added this... it's more very strange behavior.

Reply:
lgtm, but for some reason when i pull down your branch in my vs code and remove this line I am not getting the linting error to show back up